code/game/machinery/camera/camera.dm

```diff
@@ -472,8 +468,6 @@ CREATION_TEST_IGNORE_SUBTYPES(/obj/machinery/camera)
 	var/change_msg = "deactivates"
 	if(status)
 		change_msg = "reactivates"
-		triggerCameraAlarm()
-		addtimer(CALLBACK(src, PROC_REF(cancelCameraAlarm)), 100)
```

Comment:
i'm surprised these alarms exist too but it looks like it only alarms when the camera is reactivated?

Reply:
wirecutter_act seems to call toggle_camera which then calls update_camera